### PR TITLE
Fixing pre-commit errors

### PR DESCRIPTION
**Pull request by rjlohan**
*Issue #, if available:* N/A

*Description of changes:* `master` branch had issues in `pre-commit` locally. This PR cleans up the branch.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,6 +42,8 @@ def find_version(*file_paths):
         "Jinja2>=2.10",
         "jsonschema>=3.0.1",
         "pytest>=4.5.0",
+        "pytest-random-order>=1.0.4",
+        "pytest-localserver>=0.5.0",
```
Comment (miparnisari) [inline setup.py:46]: How is this related to pre-commit?

Comment (rjlohan) [reply setup.py:46]: `pytest` runs in pre-commit and wasn't passing because of missing dependencies - something in the chain must have changed.
